cargo-dist/src/tasks.rs

```diff
@@ -1729,7 +1729,7 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             warn!("skipping shell installer: not building any supported platforms (use --artifacts=global)");
             return;
         };
-        let bin_aliases = self.inner.bin_aliases.for_targets(&target_triples);
```

Comment:
where does this change come from?

Reply:
We used to look up the workspace value from the DistGraph (`self.inner`), but now we're reading it from `release`.